lib/papertrail/http_client.rb

```diff
@@ -156,6 +136,21 @@ def escape(s)
         '%' + $&.unpack('H2' * $&.bytesize).join('%').upcase
       }.tr(' ', '+')
     end
+
+    def attempt(tries=3, wait=nil)
+      return unless block_given?
```

Comment:
Since this is an internal method to this class, I think it's fine to let this method blow up when calling `yield` rather than returning if there's no block. If someone calls this method _without_ a block, it's definitely an error. I could also see raising `ArgumentError`. I don't really have a preference.